service/service.go

```diff
@@ -139,6 +141,17 @@ func New(options ...Option) (Owner, error) {
 		metrics.Freeze()
 	}
 
+	if svc.RuntimeMetricsCollector() == nil {
+		var runtimeMetricsConfig metrics.RuntimeConfig
+		err := svc.configProvider.Get("runtimeMetrics").PopulateStruct(&runtimeMetricsConfig)
```

Comment:
We manage it, we can make it whatever we want. I agree with Glib, this should be nested `metrics.runtime`